data/eos-metrics-event-recorder.service.in

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=EndlessOS Metrics Event Recorder Server
-Wants=systemd-logind.service
```

Comment:
> To be clear, there is nothing in the documentation of systemd that necessitates that systemd behave as such

Yes there is. See the documentation of http://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=